README.md

```diff
@@ -74,6 +74,9 @@ more information.
 
         $ make run
 
+5. When signing up for the app locally, you can view sent system emails at
+   http://localhost:1080/.
+
```

Comment:
oh I didn't even see that bit since I skipped over the Docker instructions. Perhaps we should combine this part of the docs somehow?